src/uct/api/uct_def.h

```diff
@@ -275,6 +288,29 @@ typedef void (*uct_unpack_callback_t)(void *arg, const void *data, size_t length
 
 
 /**
+ * @ingroup UCT_RESOURCE
+ * @brief Callback to process incoming connection establishment messages.
+ *
+ * This callback routine will be invoked upon receiving one of the @ref
+ * uct_conn_event_type_t events. It will allow the user to handle these events,
+ * read data and insert his data in a given buffer.
+ *
+ * @param [in]      event       Connection establishment event to handle.
+ * @param [in]      arg         User's argument for this callback.
+ * @param [in]      buffer_in   Points to the received data.
```

Comment:
need to write that buffer_in is the priv_data passed to connect() and buffer_out will be passed to buffer_in on client side